.github/workflows/shup_release.yml

```diff
@@ -0,0 +1,225 @@
+name: Release ShanoirUploader App
+
+on:
+  push:
+    branches:
```

Comment:
Hi @LaurentPV, thank you for this great PR.
I thought, we do not react on push, but add something like a link or button, where we can trigger a release, independent from the push action? Anthony is using this "on: workflow_dispatch:" on shanoir-deploy.
I hope this sounds logic?

Reply:
perfect